extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -373,7 +374,7 @@ function ( $ret, $item ) {
 
 	function getJsonItem( $titleName, $ns, $pageId ) {
 		$title = Title::makeTitle( $ns, $titleName );
-		list( $image_id, $image_url ) = CuratedContentSpecialController::findImageIfNotSet( 0, $pageId );
+		list( $image_id, $image_url ) = CuratedContentHelper::findImage( 0, $pageId );
```

Comment:
should it be 0 or null? I think null would show better that you are not passing imageId. Or maybe you want to find image with id 0 on article with pageId, then it's fine.